torch_uncertainty/layers/bayesian/sampler.py

```diff
@@ -43,26 +41,27 @@ def log_posterior(self, weight: Tensor | None = None) -> Tensor:
         return -lposterior.sum()
 
 
-class PriorDistribution(nn.Module):
+class CenteredGaussianMixture(nn.Module):
```

Comment:
Should inherit from torch.distributions.Distribution?

Reply:
It's not possible for now, but we'll try in the following PR...